pkgs/build-support/dotnet/add-nuget-deps/default.nix

```diff
@@ -67,6 +68,7 @@ attrs
             isExecutable = true;
             inherit cacert;
             nugetToNix = nuget-to-nix;
+            nixfmt = nixfmt-rfc-style;
```

Comment:
Not really relevant to this PR, but rather than substituting these in, would it make sense to instead set PATH using `lib.mkBinPath [ nuget-to-nix nixfmt-rfc-style ]`?